ProcessLib/Process.h

```diff
@@ -214,15 +217,23 @@ class Process
 		    static_cast<const MeshLib::NodePartitionedMesh&>(_mesh);
 		mat_opt.d_nz = pmesh.getMaximumNConnectedNodesToNode();
 		mat_opt.o_nz = mat_opt.d_nz;
+		mat_opt.is_global_size = false;
 		const std::size_t num_unknowns =
-		    _local_to_global_index_map->dofSizeGlobal();
+		    _local_to_global_index_map->dofSizeLocal();
```

Comment:
When assembling a nonlinear PDE, one also needs access to the ghost entries, but here we only have non-ghost entries. Can that be changes easily, e.g., by using `dofSize()`?